tests/utils.go

```diff
@@ -2938,10 +2938,10 @@ func LoginToVM(vmi *v1.VirtualMachineInstance, loginTo console.LoginToFactory) *
 	vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
 	ExpectWithOffset(1, err).ToNot(HaveOccurred())
 
-	// Lets make sure that the OS is up by waiting until we can login
-
-	ExpectWithOffset(1, loginTo(vmi)).To(Succeed())
-
+	if loginTo != nil {
```

Comment:
Can you please explain how is this function useful when there is no login?
Seems to me that the function should just be removed and the `console.Login*` ones be used directly.

Reply:
It just nice to have, not related / nor mandatory for this PR.
I will drop this commit from this PR, its not in the scope.
its was just in order to be able to call `WaitUntilVMIReadyWithContext` family without giving a LoginToFactory